src/main/java/uk/gov/hmcts/reform/lrdapi/domain/Region.java

```diff
@@ -41,6 +44,10 @@
     @Column(name = "updated_time")
     private LocalDateTime updatedTime;
 
+    @Column(name = "api_enabled")
+    @Fetch(SUBSELECT)
```

Comment:
you need to have `@Fetch(SUBSELECT)` in ` @OneToMany` where you use Collections. Please remove from here and add it in ` @OneToMany`

Reply:
Addressed here: https://github.com/hmcts/rd-location-ref-api/pull/273/commits/7654a02a324603c9bfc4c3154d500bb7170949d2